api/admin/model/quicksight.py

```diff
@@ -6,12 +6,12 @@
 
 
 class QuicksightGenerateUrlRequest(CustomBaseModel):
-    library_ids: List[int] = Field(
+    library_uuids: List[str] = Field(
```

Comment:
Instead of a string, you can probably make this `uuid.UUID` (see: https://docs.pydantic.dev/1.10/usage/types/#standard-library-types ), which should give us some validation on the UUID.